Fly_Swatter/fire_mode.py

```diff
@@ -15,7 +15,7 @@ def check_valdity(solution):
   else:
     return True
 
-def laser_handler(first_loc, seed = None, missile_speed = 10):
+def laser_handler(first_loc: radar.target_loc, seed: int = None, missile_speed = 10):
```

Comment:
type hinting needs to followed by a standardization and improvement of arguments ,parameters and function names